src/logprovider/instanceidprovider.hpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef INSTANCEIDPROVIDER_HPP_
+#define INSTANCEIDPROVIDER_HPP_
+
+#include <string>
+#include <vector>
+
+#include <aos/common/cloudprotocol/cloudprotocol.hpp>
+
+namespace aos::sm::logprovider {
+
+/*
+ * Provides service instances IDs.
+ */
+class InstanceIDProviderItf {
```

Comment:
move to logprovider.hpp as log provider requires it

Reply:
done